.github/workflows/release_update_docs_px_dev.yaml

```diff
@@ -0,0 +1,105 @@
+---
+name: update-docs-px-dev-on-release
+on:
+  push:
+    # TODO(ddelnano): Remove this in favor of vizier release events only once testing is complete
+    branches:
+    - test-create-docs-px-dev-prs-on-documentation-updates
+  release:
+    types: [published]
+permissions:
+  contents: read
+jobs:
+  # TODO(ddelnano): Uncomment this and retest after job is verified fully
+  get-dev-image:
+    if: startsWith(github.head_ref, 'release/vizier/')
```

Comment:
you should use `github.ref` because `head_ref` only exists on `pull_request` and `pull_request_target` events

Reply:
Updated